vllm/model_executor/layers/pooler.py

```diff
@@ -33,6 +35,9 @@ def __init__(self, pooling_type: PoolingType, normalize: bool):
 
         self.pooling_type = pooling_type
         self.normalize = normalize
+        returned_token_ids = os.environ.get('RETURNED_TOKEN_IDS', '648,387')
```

Comment:
Setting the environment variable may work as a quick hack, but it's not really maintainable. I suggest adding a new config `PoolerConfig` to pass this information via `EngineArgs` to the model (similar to how we have `MultiModalConfig`), which can be used to initialize the `Pooler` instance.

Reply:
Also see #9506

Reply:
Thank you very much for your suggestions. I have made the changes as you recommended.